org.wso2.codequalitymatrices/src/test/java/com/wso2/code/quality/metrics/GithubResponsesTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.code.quality.metrics;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * A class used to test saveRepoNames,savePrNumberAndRepoName and saveReviewers methods of the GithubResponses class
+ *
+ * @since 1.0.0
+ */
+public class GithubResponsesTest {
+    private static String githubToken;
+    private static GithubApiCaller githubApiCaller;
+
+    @BeforeClass
+    public static void setupTheEnvironment() {
+        githubToken = new Token().getGithubToken();
+        githubApiCaller = new GithubApiCaller();
+    }
+
+    @Test
+    public void testSaveRepoNames() throws CodeQualityMetricsException {
+        Map<String, List<String>> commitHashWithRepoNames = new HashMap<>();
+        commitHashWithRepoNames.put("eaa45529cbabc5f30a2ffaa4781821ad0a5223ab",
```

Comment:
These hashes seems to be used in multiple places. So might be able to use a constant.